src/dev/i18n/extract_default_translations.js

```diff
@@ -17,15 +17,17 @@
  * under the License.
  */
 
-import { resolve } from 'path';
+import path from 'path';
 import { i18n } from '@kbn/i18n';
 import JSON5 from 'json5';
+import normalize from 'normalize-path';
 
 import { extractHtmlMessages } from './extract_html_messages';
 import { extractCodeMessages } from './extract_code_messages';
 import { extractPugMessages } from './extract_pug_messages';
 import { extractHandlebarsMessages } from './extract_handlebars_messages';
 import { globAsync, makeDirAsync, accessAsync, readFileAsync, writeFileAsync } from './utils';
+import config from '../../../.localizationrc.json';
```

Comment:
__question:__ @LeanidShutau @yankouskia how do you feel about moving this reference to `run_extract_default_translations.js` and supplying it to `extractDefaultTranslations` as `config` property? Will be a little bit easier to test (e.g. no need in `jest.mock` for localizationrc.json) and integrate with other tools that may want to supply config differently?

Reply:
https://github.com/elastic/kibana/pull/21588#discussion_r208872589